Assignment9_spec.rb

```diff
@@ -0,0 +1,26 @@
+# spec/rectangle_spec.rb
+
+require_relative '../lib/rectangle'
```

Comment:
@Subhajit-QI I could not find any file lib/rectangle.rb in this PR. Can you please add it.